.danger.php

```diff
@@ -169,190 +110,92 @@ function (Context $context): void {
             $context->warning('You updated the shopware.yaml, please consider to update the config-schema.json');
         }
     })
-    ->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                $files = $context->platform->pullRequest->getFiles();
-
-                $relevant = $files->matches('src/Core/*.php')->count() > 0
-                    || $files->matches('src/Elasticsearch/*.php')->count() > 0
-                    || $files->matches('src/Storefront/Migration/')->count() > 0;
-
-                if (!$relevant) {
-                    return;
-                }
-
-                $labels = ['core__component'];
-                if ($files->matches('src/**/Cart/')->count() > 0) {
-                    $labels[] = 'core__cart';
-                }
-                if ($files->matches('src/**/*Definition.php')->count() > 0) {
-                    $labels[] = 'core__definition';
-                }
-                if ($files->matches('src/**/*Route.php')->count() > 0) {
-                    $labels[] = 'core__store-api';
-                }
-                if ($files->matches('src/**/Migration/**/Migration*.php')->count() > 0) {
-                    $labels[] = 'core__migration';
-                }
-                if ($files->matches('src/Elasticsearch/')->count() > 0) {
-                    $labels[] = 'core__elasticsearch';
-                }
-                if ($files->matches('src/**/DataAbstractionLayer/')->count() > 0) {
-                    $labels[] = 'core__dal';
-                }
-
-                $context->platform->addLabels(...$labels);
-            },
-        ]
-    ))->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                $files = $context->platform->pullRequest->getFiles();
-
-                $bcChange = $files->matches('.bc-exclude.php')->count() > 0;
-
-                if (!$bcChange) {
-                    return;
-                }
-
-                $context->platform->addLabels('bc_exclude_php');
-            },
-        ]
-    ))
-    ->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                if (!preg_match('/(?m)^((WIP:\s)|^(Draft:\s)|^(DRAFT:\s))?(\[[\w.]+]\s)?NEXT-\d*\s-\s\w/', $context->platform->pullRequest->title)) {
-                    $context->failure(sprintf('The title `%s` does not match our requirements. Example: NEXT-00000 - My Title', $context->platform->pullRequest->title));
-                }
-            },
-        ]
-    ))
-    ->useRule(new Condition(
```

Comment:
@shopware/product-operations we cannot skip something anymore, right?

Reply:
Not really

Reply:
nope